journald/lib_journald.cpp

```diff
@@ -215,8 +243,6 @@ class JournaldContainerLoggerProcess :
       labels.add_labels()->CopyFrom(label);
     }
```

Comment:
The comment above the sandbox directory parsing logic can be updated now:
```
   // Derive the AgentID from the sandbox directory, which always occurs
   // after the `slaves` directory.
   // See: src/slave/paths.hpp in the Mesos codebase for more info.
```

The `ContainerID` is no longer parsed.  And the sandbox directory no longer has to deal with MESOS-1833, because that value comes from the Agent rather than the containerizer.

Reply:
Done.